src/main/java/com/eprosima/fastdds/fastddsgen.java

```diff
@@ -624,11 +623,13 @@ public static void printHelp()
         System.out.println("template, template location must be in classpath.");
         System.out.println("\t\t" + flat_output_directory_arg + ": ignore input files relative paths and place all generated files in the specified output directory.");
         System.out.println("\t\t" + fusion_arg + ": activates fusion.");
+        System.out.print("\t\t" + flat_output_directory_arg + ": ignore input files relative paths and place all ");
```

Comment:
Duplicated

Reply:
removed in [8fc2a86](https://github.com/eProsima/Fast-DDS-Gen/pull/319/commits/8fc2a861bb7168060e6ace6f484914646c47d570)